DayrSuryan/DSEthiop23.xml

```diff
@@ -27,50 +27,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                        <locus target="#1r"/>
+                        ብአቦ፡  ስም፡  ብወዲ፡  ስምን፡  ብመንፈስ፡  ቅዱስ፡  ስምን፡ 
+                           ሐደ፡  አምላክ፡  አሜን። በስመ፡  አብ፡  ወወልድ፡  ወመንፈስ፡  ቅዱስ፡   አሐዱ፡  አምላክ፡  አሜን ።
+                           እግዚኦ፡  ነመስግነከ፡  አሎና፡  እግዚኦ፡  ነክብረካ፡  አሎና፡  እግዚኦ፡  ብአኻ፡  ንእመን፡  አሎና።
+                     </incipit>
+                     
+                     <explicit  xml:lang="ti">
+                              <locus target="#9r"/>
+                           ከምቲ፡  ነቦታትና፡  ንአብርሃምን፡  ንዘርዑን፡  ዝነገሮም፡  ክሳዕ፡  ዘለዓለም፡  ምሕረቱ፡  እናዘከረ፡  
+                                    ንእስራኤል፡  ባርያኡ፡  ተቀበሎ። (ብሥራተ፡  ሉቃ፡  1፡  46-55።)
+                                    ካብ፡  እዙይ፡  ንደኀር፡  ዘሎ፡  ከአ፡  እቲ፡  ጸሎታት፡  እዩ። ሰለ፡  እዚ፡  ባህልን፡  እምነትን፡  ፍቅሪ፡  ዓድና፤ ሎሚ፡  ግዝእቲ፡  ጽባሕ፡  ሐራ፡  እትከውን፡  
+                                    ኤርትራ፡  ከይትርስዑ፡  ኢለ፡  እነሄ፡  ጽሐፈልኩም፡  አሎኩ፡  ኤርትራዊ፡  አቦኩምን፡  ሐውኩምን፡  አባ፡  ጴጥሮስ፡  ገብረ፡  ሥላሴ።
+                           </explicit>
+                                 
                   </msItem>
+                  
 
                   <msItem xml:id="ms_i4">
-                     <locus from="66r"/>
-                     <title type="complete">Nāy ʾamānǝyān qǝdolāse</title>
-                     <textLang mainLang="ti"/>
+                     <title type="complete" ref="LIT5910QeddaseTi"/>
+                     
+                     <msItem xml:id="ms_i4.1">
+                        <locus from="12r"/>
+                        <title type="complete">Foreword</title>
+                        <textLang mainLang="en" otherLangs="ti"/>
+                        <note>Foreword in English (printed)  and Tigrinya on the composition of the Tigrinya translation of the
+                           Śǝrʿāta qǝddāse.</note>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i4.2">
+                        <locus from="1r"/>
+                        <title type="complete" ref="LIT5910QeddaseTi#SerataQ"/>
+                        <textLang mainLang="ti"/>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i4.3">
+                        <locus to="122r"/>
+                        <title type="complete" ref="LIT5910QeddaseTi#Qeddase"/>
+                        <textLang mainLang="ti"/>
+                        
+                        <msItem xml:id="ms_i4.3.1">
+                           <title type="complete">Foreword</title>
+                           <textLang mainLang="en" otherLangs="ti"/>
+                           <note>Foreword in English and Tigrinya on the two folios preceding <locus target="#66"/>.</note>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i4.3.2">
+                           <locus from="66r" to="122r"/>
+                           <title type="complete">Nāy ʾamānǝyān qǝddāse</title>
+                           <textLang mainLang="gez" otherLangs="ti"/>
```

Comment:
Translation: "Liturgy of the Faithful" (before it was liturgy for faithful + catehchumens)